Examples/Physics_applications/laser_acceleration/PICMI_inputs_3d.py

```diff
@@ -0,0 +1,121 @@
+from pywarpx import picmi
+
+# Physical constants
+c = picmi.constants.c
+q_e = picmi.constants.q_e
+
+# Number of time steps
+max_steps = 1000
```

Comment:
You might need to set a shorter value here and in 2D/RZ.
We overwrite them in `WarpX-test.ini` to 100 or 200:
https://github.com/ECP-WarpX/WarpX/blob/8f3d564d25a25b420521af7b81a004d8d49aeb71/Regression/WarpX-tests.ini#L877-L880